### DISCUSSION TITLE
Convert Markdown to HTML

### DISCUSSION THREAD

**Discussion by ClaudioBarcaTEORESI** [Stirling-Tools/Stirling-PDF#2431] [Q&A]
Yes, I know, this product has the PDF format as main target. 
But there are a lot of converters into Stirling, and to use it as central document generator into my app I should be use also the MARKDOWN to HTML format.

Is there a way to convert a markdown document into HTML page using StirlingPDF ?

**Comment by AumGupta** [comment]


**Reply by Frooodle** [reply]
There is no such Stirling-pdf import
Certainly not by us
Not to be trusted

**Comment by 173787247** [comment]
## Files / crawl / PDF («Convert Markdown to HTML»)

- **PDF + CJK**: need an embeddable font / a backend that keeps unicode; “garbled markdown” is usually extraction encoding, not your editor.
- **Crawl**: respect robots, cap depth, and don’t dump full HTML into the LLM — markdown + metadata, or you blow the context.
- **Memory**: Playwright/Chromium crawlers idle-high; reuse one browser, don’t spawn per URL.
